examples/nft-nd/nft.go

```diff
@@ -285,3 +287,59 @@ func Properties(id []byte) map[string]string {
 	}
 	return result
 }
+
+// RoyaltyRecipient contains information about the recipient and the royalty amount.
+type RoyaltyRecipient struct {
+	Address interop.Hash160
+	Amount  int
+}
+
+// RoyaltyInfo returns a list of royalty recipients and the corresponding royalty amounts.
+func RoyaltyInfo(tokenID []byte, royaltyToken interop.Hash160, salePrice int) []RoyaltyRecipient {
+	if salePrice <= 0 {
+		panic("sale price must be positive")
+	}
+
+	executingHash := runtime.GetExecutingScriptHash()
+	if !royaltyToken.Equals(executingHash) {
+		panic("invalid royalty token")
+	}
+
+	ctx := storage.GetReadOnlyContext()
+	owner := getOwnerOf(ctx, tokenID)
+	if owner == nil {
+		panic("invalid token ID")
+	}
+
+	royaltyInfoKey := append([]byte(royaltyInfoPrefix), tokenID...)
+	val := storage.Get(ctx, royaltyInfoKey)
+
+	var recipients []RoyaltyRecipient
+	if val == nil {
+		recipients = []RoyaltyRecipient{
+			{
+				Address: owner,
+				Amount:  salePrice / 10,
+			},
+		}
+	} else {
+		storedRecipients := val.([]RoyaltyRecipient)
```

Comment:
Have you tested this line? IMO, it won't work like this, because you use `std.Serialize` to put it into the storage, hence you need to use `std.Deserialize` to get it.

Reply:
Fixed it. and tested